src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/CountOperation.java

```diff
@@ -20,15 +20,32 @@
  * Operation that returns count of documents based on the filter condition. Written with the
  * assumption that all variables to be indexed.
  */
-public record CountOperation(CommandContext commandContext, LogicalExpression logicalExpression)
+public record CountOperation(
```

Comment:
Would it make sense to pass `OperationsConfig` instead of separate fields? (not sure, just wondering)

Reply:
Actually, probably better this way for use, n/m.

Reply:
All other commands like find, delete etc did this way, so following same approach.